index.d.ts

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+
+declare class ReactSwipeable<T extends Element = HTMLElement> extends React.Component<
+  ReactSwipeable.SwipeableProps<T>
+> {}
+
+interface EventData<T> {
+  event: React.TouchEvent<T>;
+  deltaX: number;
+  deltaY: number;
+  absX: number;
+  absY: number;
+  velocity: number;
+  dir: string;
```

Comment:
can we be explicit with these strings?
https://github.com/dogfessional/react-swipeable/blob/ac7239652c5912d275c5b36193cdbb3ed2b3241d/src/index.js#L18-L21